test/plugin/test_in_syslog.rb

```diff
@@ -321,11 +321,11 @@ def compare_test_result(events, tests, options = {})
   end
 
   sub_test_case 'octet counting frame' do
-    def test_msg_size_with_tcp
+    def test_octet_counting_with_tcp
       d = create_driver([CONFIG, "<transport tcp> \n</transport>", 'frame_type octet_count'].join("\n"))
       tests = create_test_case
 
-      d.run(expect_emits: 2) do
+      d.run(expect_emits: 1) do
```

Comment:
Is there a reason you changed this line? I think it shouldn't be changed since two messages are sent.